src/CodeMirror.js

```diff
@@ -0,0 +1,218 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
Two files whose case-insensitive names are the same seems like a bad idea -- some fool will want to check this out on a FAT filesystem at some point. Maybe rename codemirror.js to index.js?